pr_check.sh

```diff
@@ -6,7 +6,7 @@
 APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
 COMPONENT_NAME="ccx-notification-service"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
 IMAGE="quay.io/cloudservices/ccx-notification-service"
-COMPONENTS="ccx-data-pipeline ccx-insights-results dvo-writer ccx-smart-proxy ccx-notification-writer ccx-notification-service ccx-notification-db-cleaner notifications-backend notifications-aggregator notifications-engine insights-content-service ccx-mock-ams ccx-upgrades-sso-mock insights-content-template-renderer"  # space-separated list of components to load
+COMPONENTS="ccx-data-pipeline ccx-insights-results dvo-writer ccx-smart-proxy ccx-notification-writer ccx-notification-service ccx-notification-db-cleaner notifications-backend notifications-aggregator notifications-engine insights-content-service ccx-mock-ams ccx-upgrades-sso-mock insights-content-template-renderer ccx-redis"  # space-separated list of components to load
```

Comment:
why does the notification service need Redis?

Reply:
+smart-proxy

Reply:
It was needed for some service (Im not sure witch one but iqe tests were failing without it but i can double check)

Reply:
Not sure why it is needed, but it fixed the pr_check failure.

Reply:
Previously it was failing because bonfire could not deploy smart-proxy properly to ephemeral. So probably because of dependency on ccx-redis.

Reply:
Im going to check if the smart-proxy is needed than and if not i will update the pr

Reply:
Summary:
- smart-proxy needs ccx-redis, otherwise it won't get deployed
- iqe-tests require smart-proxy to check success of uploading a test archive.

Reply:
I also remember I had to recently add ccx-redis to some pr_checks because they were failing.